src/Compilers/CSharp/Portable/Symbols/Metadata/PE/NullableTypeDecoder.cs

```diff
@@ -16,15 +16,15 @@ class NullableTypeDecoder
             Debug.Assert((object)metadataType != null);
 
             ImmutableArray<bool> nullableTransformFlags;
-            if (containingModule.Module.HasNullableAttribute(targetSymbolToken, out nullableTransformFlags))
-            {
-                int position = 0;
-                TypeSymbolWithAnnotations result;
+            containingModule.Module.HasNullableAttribute(targetSymbolToken, out nullableTransformFlags);
```

Comment:
I'd prefer keeping the condition as long as we have this method return a bool.
If we decide at some later time to not return a bool, then we can remove the condition.

Reply:
We want to call `ApplyNullableTransforms` regardless of the return value of `HasNullableAttribute`. Is the suggestion to explicitly set `nullableTransformFlags` to `default` if `HasNullableAttribute` returns `false`?

Reply:
Ah, ok. Leave as-is.